src/main/java/org/kainos/ea/validator/CapabilityValidator.java

```diff
@@ -0,0 +1,18 @@
+package org.kainos.ea.validator;
+
+import org.kainos.ea.cli.CreateCapabilityRequest;
+
+import javax.validation.ValidationException;
+
+public class CapabilityValidator {
+
+    public void validate(CreateCapabilityRequest capabilityRequest) {
+
+        if (capabilityRequest.getCapabilityName() == null) {
+            throw new ValidationException("Capability name is null");
+        }
+        if (capabilityRequest.getCapabilityName().length() > 64) {
+            throw new ValidationException("Capability name exceeds size limit");
+        }
+    }
```

Comment:
Do you need to check the empty string? And is there a unique constraint on the DB for the capability name?

Reply:
There is no unique constraint on the table, and I have now added an empty string check.